iosApp/iosApp/ComponentViews/AlertCard.swift

```diff
@@ -38,24 +43,11 @@ struct AlertCard: View {
         }
     }
 
-    var headerString: String {
-        let formattedAlert = FormattedAlert(alert: alert)
-        return switch spec {
-        case .downstream: formattedAlert.downstreamLabel
-        case .elevator: alert.header ?? formattedAlert.effect
-        case .delay: delayHeader(formattedAlert)
-        default: formattedAlert.effect
-        }
-    }
-
-    func delayHeader(_ formattedAlert: FormattedAlert) -> String {
-        if let cause = formattedAlert.dueToCause {
-            String(format: NSLocalizedString(
-                "Delays due to %@",
-                comment: "Describes the cause of a delay. Ex: 'Delays due to [traffic]'"
-            ), cause)
-        } else {
-            NSLocalizedString("Delays", comment: "Generic delay alert label when cause is unknown")
+    var headerFont: Typography {
```

Comment:
TODO: Check this. We probably dont' want calloutSemibold if there is an alert summary to display, only if it is a fallback

Reply:
This is handled in https://github.com/mbta/mobile_app/pull/898

Reply:
Backed out this change